query-node/schemas/content.graphql

```diff
@@ -365,3 +368,42 @@ type Comment @entity {
   "The event the comment was moderated in (if any)"
   moderatedInEvent: CommentModeratedEvent @derivedFrom(field: "comment")
 }
+
+# this enum needs to correspond to runtime enum of same name
+enum ChannelActionPermission {
+  UpdateChannelMetadata
+  ManageNonVideoChannelAssets
+  ManageChannelCollaborators
+  UpdateVideoMetadata
+  AddVideo
+  ManageVideoAssets
+  DeleteChannel
+  DeleteVideo
+  ManageVideoNfts
+  AgentRemark
+  TransferChannel
+  ClaimChannelReward
+  WithdrawFromChannelBalance
+  IssueCreatorToken
+  ClaimCreatorTokenPatronage
+  InitAndManageCreatorTokenSale
+  CreatorTokenIssuerTransfer
+  MakeCreatorTokenPermissionless
+  ReduceCreatorTokenPatronageRate
+  ManageRevenueSplits
+  DeissueCreatorToken
+}
+
+type ChannelAgentPermissions @entity {
+  "Relevant channel"
+  channel: Channel!
+
+  "Related member"
+  member: Membership!
+
+  # unfortunately permissions can't be modeled as ist of enums
+  # `permissions: [ChannelActionPermission!]!`
+  # rework it after this feature is available https://github.com/Joystream/hydra/issues/507
+  "List of member's permissions"
+  permission: ChannelActionPermission!
```

Comment:
Since hydra doesn't support a list of enums, so for a missing feature I think we don't need to complicate the schema design. We can have `permissions` as a list of Strings, e.g. `permissions: [String!]!`. Though this way, we won't have Db enforced checks, I think we can mitigate this with better checks in the mappings.

Reply:
Done. No extra checks are needed because QN only displays data - all the checks are done in the runtime.